File: poh20/java/7.java
//usr/bin/env java $0 $@;exit
import java.util.*;
interface Main{static void main(String[]z){
Scanner x=new Scanner(System.in);
x.next();
char[]a=x.next().toCharArray();Arrays.sort(a);
char[]b=x.next().toCharArray();Arrays.sort(b);
System.out.println(Arrays.equals(a,b)?"Yes":"No");
}}

File: poh20/perl/7.pl
#!/usr/bin/perl
#use List::Util 'all', 'zip';
<>;
@a=sort split'',glob<>;
@b=sort split'',glob<>;
print @a~~@b?'Yes':'No'
#print((all {$_->[0] eq $_->[1]} zip(\@a,\@b))?'Yes':'No')
